packages/fury-adapter-oas3-parser/test/integration/fixtures/petstore.json

```diff
@@ -1510,7 +1510,7 @@
           ]
         }
       },
-      "content": "'Media Type Object' contains unsupported key 'schema'"
+      "content": "'Schema Object' contains invalid key '$ref'"
```

Comment:
These are currently unsupported, I'm preparing separate PR implementing reference support for this property.